src/LoRa.h

```diff
@@ -58,6 +58,7 @@ class LoRaClass : public Stream {
 #ifndef ARDUINO_SAMD_MKRWAN1300
   void onReceive(void(*callback)(int));
   void onTxDone(void(*callback)());
+  void onError(void(*callback)());
```

Comment:
I'm thinking this should be renamed to something more specific, for example: `onCrcError` or `onReceiveCrcError`. 

Thoughts?

Reply:
Please name it in accordance with the rest of the repo. No objections.

Reply:
Hi @artemen if you can address these minor changes I'll merge this PR and include in the next release, which I'd like to get to this week.

Thanks for this contribution, it will add some important insight to error conditions.